modules/imgcodecs/src/loadsave.cpp

```diff
@@ -132,14 +132,17 @@ struct ImageCodecInitializer
     */
     ImageCodecInitializer()
     {
-#ifdef HAVE_AVIF
-        decoders.push_back(makePtr<AvifDecoder>());
-        encoders.push_back(makePtr<AvifEncoder>());
-#endif
         /// BMP Support
         decoders.push_back( makePtr<BmpDecoder>() );
         encoders.push_back( makePtr<BmpEncoder>() );
 
+        /// GIF Support
```

Comment:
You need to guard that code with an `#ifdef HAVE_GIF`

Reply:
done， i set that initially on